### ISSUE TITLE
Text color of notification doesn't change

### ISSUE THREAD

**Issue by alessgerman** [WangDaYeeeeee/GeometricWeather#130]
Hi, it looks like even if I try to switch the notification's text color to something besides "Dark", it still says "Text color: Dark" on the "Notification color" settings screen, and the notification's text color doesn't change. I'm trying to make a dark-colored notification (white text on black background), but I can't do it because the text color won't change.
I'm using a Galaxy S10, Android 9.

Thanks!

**Comment by WangDaYeeeeee** [comment]
Could you show me some screenshots?

**Comment by alessgerman** [comment]
I figured out a way to make it work! I need to force-close the app before the notification text color changes stick. Otherwise it won't let me change it. It would still be nice to not have to force-close and restart the app before the text color changes stick though.

I got it to change to "light", what I wanted, but for the sake of giving an example, let's say I wanted to switch it to grey from light.
Screenshots:
Screenshot 1, I press "Grey": https://i.ibb.co/Mhb2SwB/20200414-205436.jpg
Screenshot 2, it still shows "Light", the former setting, and acts like I never pressed "Grey": https://i.ibb.co/MMpssDD/20200414-205408.jpg
